src/modules/hip/kernel/flip.hpp

```diff
@@ -308,7 +334,7 @@ RppStatus hip_exec_flip_tensor(T *srcPtr,
                                make_uint3(srcDescPtr->strides.nStride, srcDescPtr->strides.cStride, srcDescPtr->strides.hStride),
                                dstPtr,
                                make_uint2(dstDescPtr->strides.nStride, dstDescPtr->strides.hStride),
-                               make_uint2(dstDescPtr->w - 8, dstDescPtr->h),
+                               make_uint2(dstDescPtr->w, dstDescPtr->h),
```

Comment:
if ((id_y >= dstDimsWH.y) || (id_x >= dstDimsWH.x))
    {
        return;
    }
    if we do -8 then It is skipping the last 8 elements of a row, it is because of the above condition
    I reffered the CMN implementation also. In CMN there is no -8 operation involved just passing width and height